### ISSUE TITLE
FrozenPromptGenerator.generate() got an unexpected keyword argument 'seeds'

### ISSUE THREAD

**Issue by akx** [adieyal/sd-dynamic-prompts#448]
```
06:45:34-742597 ERROR    Running script process:
                         C:\automatic\extensions\sd-dynamic-prompts\scripts\dynamic_prompting.py: TypeError
┌───────────────────────────────────────── Traceback (most recent call last) ─────────────────────────────────────────┐
│ C:\automatic\modules\scripts.py:376 in process                                                                │
│                                                                                                                     │
│   375 │   │   │   │   args = p.per_script_args.get(script.title(), p.script_args[script.args_f                      │
│ > 376 │   │   │   │   script.process(p, *args, **kwargs)                                                            │
│   377 │   │   │   │   # log.debug(f'Script process end  : {script.title()}')                                        │
│                                                                                                                     │
│ C:\automatic\extensions\sd-dynamic-prompts\sd_dynamic_prompts\dynamic_prompting.py:440 in process             │
│                                                                                                                     │
│   439 │   │   │                                                                                                     │
│ > 440 │   │   │   all_prompts, all_negative_prompts = generate_prompts(                                             │
│   441 │   │   │   │   generator,                                                                                    │
│                                                                                                                     │
│ C:\automatic\extensions\sd-dynamic-prompts\sd_dynamic_prompts\helpers.py:93 in generate_prompts               │
│                                                                                                                     │
│    92 │   """                                                                                                       │
│ >  93 │   all_prompts = prompt_generator.generate(prompt, num_prompts, seeds=seeds) or [""]                         │
│    94                                                                                                               │
└─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘
TypeError: FrozenPromptGenerator.generate() got an unexpected keyword argument 'seeds'
```

_Originally posted by @crackfoo in https://github.com/adieyal/sd-dynamic-prompts/issues/441#issuecomment-1559413249_

**Comment by akx** [comment]
This should be fixed by https://github.com/adieyal/dynamicprompts/commit/4c3adf5ddf4c1ec9cf2c7784326f60564eaf62f7

**Comment by crackfoo** [comment]
looks like this is all resolve now. thanks!

**Comment by medledan** [comment]
> This should be fixed by [adieyal/dynamicprompts@4c3adf5](https://github.com/adieyal/dynamicprompts/commit/4c3adf5ddf4c1ec9cf2c7784326f60564eaf62f7)

This issue is resolved. Thank you so much for your help, and making this amazing extension. Appreciate it ❤️

**Comment by adieyal** [comment]
:-)
